launch/tier4_simulator_launch/launch/simulator.launch.xml

```diff
@@ -6,7 +6,7 @@
   <arg name="occupancy_grid_map_updater"/>
   <arg name="occupancy_grid_map_updater_param_path"/>
   <arg name="pose_initializer_param_path"/>
-
+  <arg name="launch_simulator_perception_modules" default="true"/>
```

Comment:
To me, this option makes the launch file messy. I wanna look for a better way. If there's nothing, I'm fine with this PR's change.

Reply:
Thank you for the review @takayuki5168. I am going to think about a better way coming days. If I find another way, I will create another PR to make launch file tidy.